samples/snippets/pom.xml

```diff
@@ -2,14 +2,14 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.google.cloud</groupId>
-  <artifactId>webrisk-snippets</artifactId>
+  <artifactId>-snippets</artifactId>
```

Comment:
This is incorrect. Can revert deleting the pom here - so that OwlBot doesn't mess this up. The pom is still required since the snippets folder has the generated samples. So let's not touch the pom